### PR TITLE
Add BUSD implicit pair to CELO/USD oracles.

### PR DESCRIPTION
**Pull request by pedro-clabs**
### Description

Points the oracle deployment to a new image supporting the BUSDUSD pair, and configures the CELO/USD oracles to use a (Binance, CELOBUSD)x(Coinbase, BUSDUSD) implicit pair as a price source.

### Tested

Deployed to Alfajores.

### Backwards compatibility

New oracle version only adds a new configuration value, and doesn't remove any public interface.

### CODE REVIEW

**Review by martinvol — APPROVED** [review]

File: .env.rc1 [hunk 1]
```diff
@@ -60,7 +60,7 @@ CELOCLI_STANDALONE_IMAGE_TAG="0.0.42"
 MOCK_ORACLE_CRON_SCHEDULE="*/5 * * * *"
 
 ORACLE_DOCKER_IMAGE_REPOSITORY="celoprod.azurecr.io/oracle"
-ORACLE_DOCKER_IMAGE_TAG="3cc6459db821f03ce0df12d2891b3ff4b34e3f05"
+ORACLE_DOCKER_IMAGE_TAG="975d290fce7098a7e6026edd964835ec495026ef"
```
Comment (martinvol) [inline .env.rc1]: I usually try to not update mainnet until tested for security

Comment (martinvol) [inline .env.rc1]: I usually try to not update mainnet until tested for security

File: .env.alfajores [hunk 2]
```diff
@@ -35,7 +35,7 @@ CELOSTATS_BANNED_ADDRESSES=""
 CELOSTATS_RESERVED_ADDRESSES=""
 
 ORACLE_DOCKER_IMAGE_REPOSITORY="celoprod.azurecr.io/oracle"
-ORACLE_DOCKER_IMAGE_TAG="3cc6459db821f03ce0df12d2891b3ff4b34e3f05"
+ORACLE_DOCKER_IMAGE_TAG="975d290fce7098a7e6026edd964835ec495026ef"
```
Comment (martinvol) [inline .env.alfajores]:
```suggestion
ORACLE_DOCKER_IMAGE_TAG="88189be0901fb17598ee57301d13ab3077ec4731"
```
```suggestion
ORACLE_DOCKER_IMAGE_TAG="975d290fce7098a7e6026edd964835ec495026ef"
```

Comment (martinvol) [inline .env.alfajores]:
```suggestion
ORACLE_DOCKER_IMAGE_TAG="88189be0901fb17598ee57301d13ab3077ec4731"
```
```suggestion
ORACLE_DOCKER_IMAGE_TAG="975d290fce7098a7e6026edd964835ec495026ef"
```

File: .env.baklava [hunk 3]
```diff
@@ -42,7 +42,7 @@ CELOCLI_STANDALONE_IMAGE_REPOSITORY="gcr.io/celo-testnet/celocli-standalone"
 CELOCLI_STANDALONE_IMAGE_TAG="0.0.30-beta2"
 
 ORACLE_DOCKER_IMAGE_REPOSITORY="celoprod.azurecr.io/oracle"
-ORACLE_DOCKER_IMAGE_TAG="3cc6459db821f03ce0df12d2891b3ff4b34e3f05"
+ORACLE_DOCKER_IMAGE_TAG="975d290fce7098a7e6026edd964835ec495026ef"
```
Comment (martinvol) [inline .env.baklava]:
```suggestion
ORACLE_DOCKER_IMAGE_TAG="88189be0901fb17598ee57301d13ab3077ec4731"
```
```suggestion
ORACLE_DOCKER_IMAGE_TAG="975d290fce7098a7e6026edd964835ec495026ef"
```

Comment (martinvol) [inline .env.baklava]:
```suggestion
ORACLE_DOCKER_IMAGE_TAG="88189be0901fb17598ee57301d13ab3077ec4731"
```
```suggestion
ORACLE_DOCKER_IMAGE_TAG="975d290fce7098a7e6026edd964835ec495026ef"
```

File: .env.rc1 (hunk 1 above)
Comment (martinvol) [inline .env.rc1]:
```suggestion
ORACLE_DOCKER_IMAGE_TAG="88189be0901fb17598ee57301d13ab3077ec4731"
```
```suggestion
ORACLE_DOCKER_IMAGE_TAG="975d290fce7098a7e6026edd964835ec495026ef"
```

Comment (martinvol) [inline .env.rc1]:
```suggestion
ORACLE_DOCKER_IMAGE_TAG="88189be0901fb17598ee57301d13ab3077ec4731"
```
```suggestion
ORACLE_DOCKER_IMAGE_TAG="975d290fce7098a7e6026edd964835ec495026ef"
```

Comment (pedro-clabs) [reply .env.rc1]: Do you mean some particular security testing? If not, I'd only merge this PR after it's tested on Alfajores/Baklava (in fact, after it's already deployed to mainnet).

File: .env.alfajores [hunk 4]
```diff
@@ -35,7 +35,7 @@ CELOSTATS_BANNED_ADDRESSES=""
 CELOSTATS_RESERVED_ADDRESSES=""
 
 ORACLE_DOCKER_IMAGE_REPOSITORY="celoprod.azurecr.io/oracle"
-ORACLE_DOCKER_IMAGE_TAG="3cc6459db821f03ce0df12d2891b3ff4b34e3f05"
+ORACLE_DOCKER_IMAGE_TAG="88189be0901fb17598ee57301d13ab3077ec4731"
```
Comment (pedro-clabs) [inline .env.alfajores:38]: @martinvol Is this update to get an oracle version with the updated Coinbase's liveness check?

Comment (martinvol) [reply .env.alfajores:38]: Yes

Comment (pedro-clabs) [reply .env.alfajores:38]: Got it, thanks. I've updated all the env files.

File: .env.rc1 (hunk 1 above)
Comment (pedro-clabs) [reply .env.rc1]: Resolving as it's already deployed to mainnet.

Comment (pedro-clabs) [reply .env.rc1]: Resolving, as it's deployed to mainnet already.

**Pull request merged**